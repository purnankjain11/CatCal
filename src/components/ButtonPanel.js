import React from 'react';
import { Text, View } from 'react-native';

export default class ButtonPanel extends React.Component {
  render() {
    return(
      <View />
    );
  }
};