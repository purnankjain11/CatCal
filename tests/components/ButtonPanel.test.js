import React from 'react';
import ButtonPanel from '../../src/components/ButtonPanel';

import renderer from 'react-test-renderer';

describe('component snapshot test', () => {
  it('renders ButtonPanel without crashing', () => {
    const rendered = renderer.create(<ButtonPanel />).toJSON();
    expect(rendered).toMatchSnapshot(); 
  });
});