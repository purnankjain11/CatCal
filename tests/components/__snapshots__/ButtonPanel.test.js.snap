// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component snapshot test renders ButtonPanel without crashing 1`] = `<View />`;
